operator/v1/types_ingress.go

```diff
@@ -678,19 +679,27 @@ type IBMLoadBalancerParameters struct {
 // OpenStackLoadBalancerParameters provides configuration settings that are
 // specific to OpenStack load balancers.
 type OpenStackLoadBalancerParameters struct {
-	// loadBalancerIP specifies the floating IP address that the load balancer will use.
+	// loadBalancerIP is tombstoned since the field was replaced by floatingIP
+	// LoadBalancerIP string `json:"loadBalancerIP,omitempty"`
+
+	// floatingIP specifies the IP address that the load balancer will use.
 	// When not specified, an IP address will be assigned randomly by the OpenStack cloud provider.
+	// When specified, the floating IP has to be pre-created otherwise the OpenStack cloud provider won't be
+	// able to create it.
```

Comment:
With an invalid or already claimed address, provisioning will fail, right?
```suggestion
	// When specified, the floating IP has to be pre-created.  If the
	// specified value is not a floating IP or is already claimed, the
	// OpenStack cloud provider won't be able to provision the load
	// balancer.
```